admin/angel/Core.c

```diff
@@ -189,14 +200,33 @@ static void initTunfd(Dict* args, void* vcontext, String* txid, struct Allocator
         struct Pipe* p = Pipe_forFiles(fileno, fileno, ctx->base, &jmp.handler, ctx->alloc);
         p->logger = ctx->logger;
         if (type == FileNo_Type_ANDROID) {
```

Comment:
@cjdelisle Do you mean to move `FileNo_Type_ANDROID` vs. `FileNo_Type_NORMAL` from Core as well?